data/wanderer/wanderers start.txt

```diff
@@ -179,67 +183,388 @@ mission "Wanderers: Hai Diplomat"
+				accept
+
+
+mission "Wanderers: Professor Available"
+	landing
+	name "Wanderers: Professor Andelena"
+	description "Fly to <planet> to pick up a professor and grad student who are willing to translate for you."
+	source
+		not planet Hai-home
+		not planet Allhome
+		not attributes uninhabited
+		government Hai Republic Syndicate "Free Worlds" Neutral Independent
+	destination Allhome
+	to offer
+		has "event: A Hai professor is available to translate"
+		not "Wanderers: Diplomacy: done"
+	on offer
+		log `Professor Andelena in the Department of Skiourology at Mongongo University has sent me a message asking for transport services on her expedition. This may be an opportunity to get an interpreter for the aliens to the north.`
+		conversation
+			`Your ship's terminal bleats out a pinging noise, accompanied by a message from a university on <planet>. "Professor Andelena in the Department of Skiourology at Mongongo University awaits your transport services on her expedition." This may be the interpreter you've been waiting for.`
+				accept
+
+
+
+mission "Wanderers: Meet Professor"
+	invisible
+	landing
+	name "Wanderers: Professor Andelena"
+	source Allhome
+	to offer
+		has "Wanderers: Professor Available: done"
+		not "Wanderers: Sayari Available After Hai Reveal: offered"
+	to complete
+		has "Wanderers: Diplomacy (Before Hai Reveal): offered"
+	on offer
+		log "Minor People" "Professor Andelena" `Professor Maindell Stormchild Andelena, IV speaks with Shroud cloud city accent and is a daughter of the rich Andelena family. Her wealthy meanderings have led her to a university on Allhome where she studies Skiorology, the Hai-studying equivalent of Anthropology.`
+		log "Minor People" "Eenap" `Professor Andelena's small Hai grad student, who is interested in studying the aliens to the north as her thesis research subject.`
+		conversation
+			`It turns out Mongongo University is the size of a large town, with its own spaceport. The Department of Skiourology has five entire buildings. You enter the largest and inquire about Professor Andalena, whose office is through a labyrinth of corridors. After a few minutes of navigation, you begin to conclude you've been trapped in a Mobius strip.`
+			`	"Captain <last>," squeaks a Hai voice near you. "I'm Eenap, Professor Andelena's grad student. Please follow me to her office so we may discuss an expedition." Soon, you arrive at Professor Andelena's office, which looks like a small museum exhibit, filled with taxidermy statues of evolutionary ancestors of the Hai, and diagrams of development of Hai society.`
+			`	"Welcome, Captain <last>! I am Professor Maindell Stormchild Andelena, IV." This human woman speaks with the famously-dignified Shroud cloud city accent. "We were planning an expedition to study planets suspected of being part of a past Hai empire, but your discovery has piqued our interest."`
```

Comment:
```suggestion
			`	"Welcome, Captain <last>! I am Professor Maindell Stormchild Andelena IV." This human woman speaks with the famously-dignified Shroud cloud city accent. "We were planning an expedition to study planets suspected of being part of a past Hai empire, but your discovery has piqued our interest."`
```
Our? Is that because of Eenap?

Reply:
In an unusual move for a professor, she considers her grad student to be a person whose wants and needs matter.